src/Controllers/API/DeleteCourseController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Portal\Controllers\API;
+
+use Portal\Models\ApplicantModel;
+use Portal\Models\CourseModel;
+use Psr\Http\Message\RequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+use Portal\Abstracts\Controller;
+
+class DeleteCourseController extends Controller
+{
+    private CourseModel $courseModel;
+
+    public function __construct(ApplicantModel $applicantModel, CourseModel $courseModel)
+    {
+        $this->courseModel = $courseModel;
+    }
+
+    public function __invoke(Request $request, Response $response, array $args): Response
+    {
+        $students = $request->getParsedBody();
+        $courseToDelete = $args['id'];
+        foreach ($students as $studentId => $value) {
+            if ($value === '0') {
```

Comment:
Could you add a comment to explain what this 0 value represents?

Reply:
If no course is assigned to an applicant, it gives the applicant the value of 0 and they are removed from all courses